scripts/script-tag.js

```diff
@@ -255,7 +260,7 @@
         const sku = id;
 
         jQueryBopis('#hc-store-card').remove();
-        eventTarget.prop("disabled", true);
+        if (event) eventTarget.prop("disabled", true);
```

Comment:
Add information about this change in PR description

Reply:
Sir, added description in PR.